packages/ubuntu_bootstrap/integration_test/ubuntu_bootstrap_test.dart

```diff
@@ -208,31 +209,31 @@ void main() {
     await tester.tapNext();
     await tester.pumpAndSettle();
 
-    await tester.testStoragePage(
-      type: StorageType.erase,
-      guidedCapability: GuidedCapability.CORE_BOOT_ENCRYPTED,
-    );
-    await tester.tapNext();
-    await tester.pumpAndSettle();
+    //await tester.testStoragePage(
+    //  type: StorageType.erase,
+    //  guidedCapability: GuidedCapability.CORE_BOOT_ENCRYPTED,
+    //);
+    //await tester.tapNext();
+    //await tester.pumpAndSettle();
 
-    await tester.testRecoveryKeyPage();
-    await tester.tapNext();
-    await tester.pumpAndSettle();
+    //await tester.testRecoveryKeyPage();
+    //await tester.tapNext();
+    //await tester.pumpAndSettle();
 
-    await tester.testConfirmPage();
-    await tester.tapConfirm();
-    await tester.pumpAndSettle();
+    //await tester.testConfirmPage();
+    //await tester.tapConfirm();
+    //await tester.pumpAndSettle();
 
-    await tester.testInstallPage();
-    await tester.pumpAndSettle();
+    //await tester.testInstallPage();
+    //await tester.pumpAndSettle();
 
-    final windowClosed = YaruTestWindow.waitForClosed();
-    await tester.tapContinueTesting();
-    await expectLater(windowClosed, completes);
+    //final windowClosed = YaruTestWindow.waitForClosed();
+    //await tester.tapContinueTesting();
+    //await expectLater(windowClosed, completes);
 
-    await verifySubiquityConfig(
-      capability: GuidedCapability.CORE_BOOT_ENCRYPTED,
-    );
+    //await verifySubiquityConfig(
+    //  capability: GuidedCapability.CORE_BOOT_ENCRYPTED,
+    //);
```

Comment:
I suspect you didn't mean to commit this

Reply:
Hahah ooops, fixed now!